recipes/torchsde/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "torchsde" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google-research/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 4ced504b8fab8963fae6d82a34ae2a51179b51b0f303f28277958272d0ce9489
+
+build:
+  number: 0
+  #script: "{{ PYTHON }} -m pip install . -vv"
```

Can you make your package installable, but not testable, without pytorch?

If you can, then maybe we just merge as is, and let users install pytorch with the recommended way.
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```
If you can, then

---

Currently they raise an exception if `pytorch` isn't available at build time:
```python
try:
    import torch
    from torch.utils import cpp_extension
except ModuleNotFoundError:
    raise ModuleNotFoundError("Unable to import torch. Please install torch>=1.6.0 at https://pytorch.org.")
```
https://github.com/google-research/torchsde/blob/9d1d65d35343be283eb6b405f7f7f0c8b0e7b106/setup.py#L20-L24

...however it look like this is changing so the easiest thing to do here is to wait for the next release (which should be happening shortly)
https://github.com/google-research/torchsde/commit/11288210e7ff30b5cb5afb3717ae967e99f9aa57